bip-0341.mediawiki

```diff
@@ -66,8 +66,10 @@ The following rules only apply when such an output is being spent. Any other out
 * If there are at least two witness elements left, script path spending is used:
 ** Call the second-to-last stack element ''s'', the script.
 ** The last stack element is called the control block ''c'', and must have length ''33 + 32m'', for a value of ''m'' that is an integer between 0 and 128<ref>'''Why is the Merkle path length limited to 128?''' The optimally space-efficient Merkle tree can be constructed based on the probabilities of the scripts in the leaves, using the Huffman algorithm. This algorithm will construct branches with lengths approximately equal to ''log<sub>2</sub>(1/probability)'', but to have branches longer than 128 you would need to have scripts with an execution chance below 1 in ''2<sup>128</sup>''. As that is our security bound, scripts that truly have such a low chance can probably be removed entirely.</ref>, inclusive. Fail if it does not have such a length.
+** Let ''c[0]'' be the ''control byte'' consisting of:
```

Comment:
```suggestion
** Let ''c[0]'' be the ''control byte''.
```
"Consisting of" doesn't go well with the bullet points that follow.

Reply:
How about:

> Let _c[0]_ be the _control byte_ consisting of fields _v_ and _pyparity_:

Reply:
That doesn't work because _v_ is not a subfield of the control byte but the entire control byte with the lowest bit masked off.

Reply:
Ok, I see what you mean.

> Let _c[0]_ be the _control byte_ encoding _v_ and _pyparity_: